samples/msgext-northwind-inventory-csharp/appsettings.TestTool.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
The test tool isn't used, so this file can be removed.

Reply:
fixed